Training/1.0 Introduction.md

```diff
@@ -12,37 +12,61 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+String - 'Hello World!'
+Int - 5
+Float - 1.1
+Bool - True or False
+List - [1,2,3,4,5]
+Tuple - (1,2,3,4,5)
+Dictionary - {'apple':20, 'pear':10}
 ```
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
 ```
-
+list
+lis = [1,2,3,4]
+arrays
+ar = np.arrays([[1,2],[3,4]])
+tuples
+tup = (1,2,3,4)
 ```
 
 ### Task 3: 
 
 We are also interested in becoming good software developers so we will need to use conditional, loops. Write an statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
 
 ```
-
+if x>2:
+x = x+1
+else print('x is less than 2')
 ```
 
 Now incorporate a for loop to calculate the value of n(x+1) for n repetitions where n=3. Store each value of the for loop in an array named y.  
 
 ```
-
+y = []
```

Comment:
This does not create an array. The correct way to create an array would be to use numpy:
`import numpy`
`y = numpy.array([])`